.github/workflows/build.yml

```diff
@@ -21,6 +21,9 @@ jobs:
 
         - name: Setup Gradle
           uses: gradle/gradle-build-action@v2
+          with:
+            gradle-version: 7.2
```

Comment:
Not sure why but GH Actions for [gradle builds](https://github.com/gradle/gradle-build-action) was starting to use 8.0 even though 7.2 was specified in the gradle-wrapper.properties file. Since 8.0 has some breaking changes from 7.2, the build was failing. Using a fixed gradle version in the GH Actions fixed the build.